ffmpeg.c

```diff
@@ -199,6 +239,13 @@ void ffmpeg_init(void){
     avcodec_register_all();
     avformat_network_init();
     av_log_set_callback((void *)ffmpeg_avcodec_log);
+
+    ret = av_lockmgr_register(ffmpeg_lockmgr_cb);
+    if (ret < 0)
+    {
+        MOTION_LOG(EMG, TYPE_ALL, SHOW_ERRNO, "%s: av_lockmgr_register failed (%d)", ret);
+        abort();
```

Comment:
Seems pretty drastic to just abend on them.  Can we get a return code and have a clean exit?